### DISCUSSION TITLE
更新包需要重启才生效

### DISCUSSION THREAD

**Discussion by crazyair** [umijs/umi#12105] [Q&A]
比如安装了 `a:0.0.1`，加了个功能更新到 `a:0.0.2`，刷新页面看不到效果，需要重新 `npm start` 才可以，怎么样能加白名单之类的，让指定的包也能更新？

**Comment by fz6m** [comment]
如果更新了依赖，需要重启项目才能重新加载，因为之前用的那份依赖已经载入到内存里去了，比如你启动了这个项目然后把 `node_modules` 全删了还能跑是一样的，旧的文件都载入到内存里去了。
